packages/fxa-settings/src/pages/Signup/ConfirmSignupCode/index.test.tsx

```diff
@@ -38,6 +46,8 @@ jest.mock('../../../lib/glean', () => ({
   default: { signupConfirmation: { view: jest.fn(), submit: jest.fn() } },
 }));
 
+jest.mock('../../../lib/hooks/useRedirect');
```

Comment:
Is this necessary? I don't seeanything being mocked here? Is it so that you can mock useRedirect later?

Reply:
Yeah, further down I'm mocking what it returns. I'm pretty sure it's necessary but I'll double check.